libs/server/Resp/AdminCommands.cs

```diff
@@ -1023,10 +1034,19 @@ private bool NetworkRegisterCs(int count, byte* ptr, CustomCommandManager custom
                     // Check optional parameters for previous sub-command
                     if (optionalParamsRead == 0 && args is RegisterCmdArgs cmdArgs)
```

Comment:
optionalParamsRead == 0 will only allow 1 optional parameter to be read, we should allow 2